### PR TITLE
Two additional options for kind of dish

### LINKED ISSUE #288: Change "Kind of dish" option to multi-options instead of a single option
Today I noticed this new feature (very nice addition) but I also noticed that you can only pick one of the options while some dishes might feature options. For example when making an AVG meal one might make vega meat for the vega's and real meat for the normal people.

So my suggestion is to impliment a checkbox there (instead of a single option) (or to change the name into "Vegetarian suitable").

### PR DESCRIPTION
**Pull request by mhvis**
I like the two extra options. Now I have it like this: 9baa1aebe11cda99df5a0a1b0eb96895a7e8dd08

Closes #288

### CODE REVIEW

File: dining/models.py
Comment (Ndoll1) [inline dining/models.py:1]: Please add 🍗 for consistency

**Review by Ndoll1 — CHANGES_REQUESTED** [review]
Please add 🍗 for consistency at "With meat"

Maybe also at  
🍗Meat or 🥕 Vegetarian
🍗Meat or 🌿 Vegan

Would still prefer to instead of "🍗Meat or 🥕 Vegetarian" and "🍗Meat or 🌿 Vegan". It would give an option as optional checkbox
🥕 Vegetarian welcome/friendly
🌿 Vegan welcome/friendly

**Review by Ndoll1 — APPROVED** [review]

**Comment by mhvis** [comment]
:+1: Added the meat icon. I don't want too much of the icons because they're not very accurate actually :laughing:. Since a carrot is both vegetarian and vegan, and meat isn't necessarily a poultry leg.

**Comment by mhvis** [comment]
> Would still prefer to instead of "🍗Meat or 🥕 Vegetarian" and "🍗Meat or 🌿 Vegan". It would give an option as optional checkbox 🥕 Vegetarian welcome/friendly 🌿 Vegan welcome/friendly

This suggests to me more like the meal is with meat by default but has a vegetarian option. Instead of you can choose between meat and vegetarian. That's also fine by me if that's a better way of doing it. But I want someone else to agree that this is better before I change it, otherwise lets leave it like this.

**Pull request merged**